test/common/upstream/health_checker_impl_test.cc

```diff
@@ -600,7 +633,7 @@ class HttpHealthCheckerImplTest : public Event::TestUsingSimulatedTime,
   }
 
   void respond(size_t index, const std::string& code, bool conn_close, bool proxy_close = false,
-               bool body = false, bool trailers = false,
+               bool body = false, absl::string_view response_body = "", bool trailers = false,
```

Comment:
You can overload the `respond` method and add the `response_body` parameter to avoid changing every call site that does not need to set the `response_body`

Reply:
Create a new method to send body and send multiple small portions.